src/basic-kokkos/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+# and RAJA Performance Suite project contributors.
+# See the RAJAPerf/COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+include_directories(SYSTEM ${CMAKE_CURRENT_SOURCE_DIR}/../basic)
```

Comment:
Please use the `INCLUDES` argument of `blt_add_library`, rather than this `include_directories` call.

https://llnl-blt.readthedocs.io/en/develop/api/target.html#blt-add-library